### PR TITLE
office-js-preview Page-to-Dialog messaging APIs

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x ] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.microsoft.com/office/dev/add-ins/develop/parent-to-dialog

### CODE REVIEW

**Comment by amcasey** [comment]
Was there supposed to be a corresponding version bump?

**Comment by Rick-Kirkham** [comment]
@amcasey Nope. No version change.

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -891,6 +891,23 @@ declare namespace Office {
...
+         * @beta
+         *
+         * @remarks
+         * 
+         * **Requirement set**: TBD
+         *
+         * You can add multiple event handlers for the specified eventType as long as the name of each event handler function is unique.
+         *
+         * @param eventType Specifies the type of event to add. This must be `Office.EventType.DialogParentMessageReceived`.
+         * @param handler The event handler function to add, whose only parameter is of type {@link Office.DialogParentMessageReceivedEventArgs}
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * @param handler The event handler function to add, whose only parameter is of type {@link Office.DialogParentMessageReceivedEventArgs}.
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -891,6 +891,23 @@ declare namespace Office {
...
+         * @param options Provides an option for preserving context data of any type, unchanged, for use in a callback.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * @param options Optional. Provides an option for preserving context data of any type, unchanged, for use in a callback.
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -1664,6 +1681,18 @@ declare namespace Office {
...
+        /**
+         * Delivers a message from the host page, such as a task pane or a UI-less function file, to a dialog that was opened from the page. 
+         * 
+         * @beta
+         * 
+         * @remarks
+         * 
+         * **Requirement set**: TBD
+         * 
+         * @param message Accepts a message from the host page to deliver to the dialog. Anything that can serialized to a string including JSON and XML can be sent.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * @param message Accepts a message from the host page to deliver to the dialog. Anything that can be serialized to a string, including JSON and XML, can be sent.
```

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -891,6 +891,23 @@ declare namespace Office {
...
+        /**
+         * Adds an event handler to the object using the specified event type.
+         * 
+         * @beta
+         *
+         * @remarks
+         * 
+         * **Requirement set**: TBD
+         *
+         * You can add multiple event handlers for the specified eventType as long as the name of each event handler function is unique.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts]:
```suggestion
         * You can add multiple event handlers for the specified event type as long as the name of each event handler function is unique.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**